src/MaaCore/Config/Miscellaneous/BattleDataConfig.h

```diff
@@ -24,6 +24,42 @@ namespace asst
             return iter->second.id;
         }
 
+        const std::string get_en(const std::string& name) const
+        { 
+            auto iter = m_chars.find(name);
+            if (iter == m_chars.cend()) {
+                return "";
```

Comment:
Open to suggestions on what's supposed to go here...

Reply:
Return the name directly?

Reply:
I've just copied how the getter was for the ID... Didn't really give much thoughts about it